packages/argon2-cffi/test_argon2-cffi.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["argon2-cffi"])
+def argon2_cffi_helper(selenium):
+    import argon2
+
+    ph = argon2.PasswordHasher(parallelism=1)
+    hash = ph.hash("test")
+
+    assert ph.verify(hash, "test") is True
+
+    with pytest.raises(argon2.exceptions.UnsupportedParamsError) as exinfo:
```

Comment:
You can't see the global scope from inside the `@run_in_pyodide` function so you'll need to import `pytest` inside the function rather than at top level.

Reply:
Thanks i added this in the next commit